### PR TITLE
Make temperature path configurable

### PR DESCRIPTION
**Pull request by streeve**
If a reasonable idea, we would need to update all the example input files and maybe do the same for substrates

### CODE REVIEW

File: examples/Inp_SimpleRaster.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@ File of grain orientations:GrainOrientationVectors_Robert.csv
 Heat transport data mesh size (microns): 5
 Time step (microseconds): 0.08333333
 Substrate grain spacing (in microns): 25
+Path to temperature file(s): examples/Temperatures
 Temperature filename: Master-Raster-150um.txt
```
Comment (streeve) [inline examples/Inp_SimpleRaster.txt:19]: Add the same note as the AMbench file?

File: examples/Inp_AMBenchMultilayer.txt [hunk 2]
```diff
@@ -2,18 +2,21 @@ Test problem using alternating odd-even layers of a cross-hatching scan pattern
 *****
 Problem type: R
 Decomposition strategy (1 for 1D domain decomposition, 2 for simple 2D decomposition, 3 for complex 2D decomposition): 1
+Path to material file: Materials
```
Comment (streeve) [inline examples/Inp_AMBenchMultilayer.txt:5]: Will it ever make sense to have more than one file? We could switch the existing input from file name to full path (except temperature, unless there's a good way to do the file glob with c++)

**Comment by streeve** [comment]
@MattRolchigo good news, I got this to work locally; bad news, I added a bug somewhere and the runs never finish (stuck at constant liquid cells)

**Comment by MattRolchigo** [comment]
@streeve most likely that's a file path issue - I don't think I have guards on the grain orientation or material file for the case if they can't be opened, so ExaCA thinks it read A/B/C/D/GrainOrientationVectors, which is leading to non-physical grain orientations/growth kinetics

**Comment by streeve** [comment]
> @streeve most likely that's a file path issue - I don't think I have guards on the grain orientation or material file for the case if they can't be opened, so ExaCA thinks it read A/B/C/D/GrainOrientationVectors, which is leading to non-physical grain orientations/growth kinetics

There are checks now for each file open in the last few commits. I'll take another look when I can

**Review by MattRolchigo — APPROVED** [review]

**Comment by mikejwg** [comment]
Could you share the eternal data in the temperature_path for testing the Example problems only with external data. By the way, when I run the DirSolidification.txt,  some strange grain orientations were found in the solidification front.

**Comment by streeve** [comment]
@mikejwg please open a new issue for this - this pull request has been merged and closed.

**Pull request merged**